src/_lib/sass/src/_modals.scss

```diff
@@ -65,22 +65,41 @@
             border-top-color: get-color("gray", 5x-light);
         }
     }
+
+    &.modal-media {
+        
+        .btn-close {
+            transform: translateX(100%); 
+            position: absolute; 
+            right: -8px; 
+            opacity: 0.5;
+            top: 0; 
+        }
+
+        .modal-content {
+            border: 0;
+            background-color: transparent;
```

Comment:
I'm not sure this should be transparent. If the img we display is a png with transparency it looks strange, e.g. https://www.freelogodesign.org/Content/img/logo-ex-7.png